src/components/Button/index.tsx

```diff
@@ -24,13 +24,13 @@ const StyledContent = styled.div`
   pointer-events: none;
 `
 
-// It will remove animation between Link and left icon that is implemented into Link component
+// Remove the animation of the Link
 const StyledLink = styled(Link)`
-  gap: 0;
-
   &:hover,
   &:focus {
-    gap: 0;
+    svg {
```

Comment:
Can we target the Icon directly ?

Reply:
It works if we export and use the `StyledIcon` from the `Icon` component, but not if we use the `Icon` component directly (maybe because it's wrapped in a `forwardRef`?)

<img width="691" alt="Screenshot 2022-12-20 at 09 55 41" src="https://user-images.githubusercontent.com/43268759/208625333-6d562966-a4ed-4c9c-bfb3-7fdc65488c6d.png">
<img width="429" alt="Screenshot 2022-12-20 at 09 55 51" src="https://user-images.githubusercontent.com/43268759/208625359-5b8256c9-0f62-4389-9c2c-54e92dbf4f62.png">

Reply:
What about 

```jsx
const StyledIcon = styled(Icon)``

const StyledLink = styled(Link)`
  &:hover,
  &:focus {
    ${StyledIcon} {
      transform: none;
    }
  }
`
```

Reply:
Just tested and it doesn't work, maybe because the style of this empty `StyledIcon` isn't the same as the one inside `Link`

Reply:
Well you'll also need to use StyledIcon in Link 😄

Reply:
Oh I didn't saw the difference between the styled of icon and styled of svg. Using a `styled.svg` in both places still doesn't work btw

Reply:
As discussed we'll keep the `svg`